compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1975,4 +1975,7 @@ error.seq.var.cannot.be.used.in.required.arg=\
   sequence variable cannot be used in a required argument
 
 error.seq.var.used.in.single.element.list.ctr.or.func.invocation=\
-  sequence variable can be used in a single element list constructor or function invocation
\ No newline at end of file
+  sequence variable can be used in a single element list constructor or function invocation
```

Comment:
Shall we change this error message? when we say "can be" it may convey the idea it is not mandatory